Helper.cs

```diff
@@ -0,0 +1,64 @@
+﻿
+public class Helper
```

Comment:
Helper class reiktų atskirai laikyti, bus mažiau makalynės:)

Reply:
But it is in different file, 😕